modules/core/test/v2/unit/coins/stx.ts

```diff
@@ -53,7 +53,8 @@ describe('STX:', function() {
     explain.outputAmount.should.equal(testData.txExplainedTransfer.outputAmount);
     explain.outputs[0].amount.should.equal(testData.txExplainedTransfer.outputAmount);
     explain.outputs[0].address.should.equal(testData.txExplainedTransfer.recipient);
-    explain.outputs[0].memo.should.equal(testData.txExplainedTransfer.memo);
+    // TODO: uncomment the following line after account-lib 2.9.0 is released
```

Comment:
I don't quite follow here - why are we commenting this out pending a new version being released? Because this repo is managed with `lerna`, the version of `account-lib` should be exactly the one that exists in this repo (`node_modules/@bitgo/account-lib` should be a symlink to `modules/account-lib`) and not the npm version when installing using `yarn`.

Reply:
What is the fix in account-lib 2.9.0 that should make this test pass?